src/vulkan/descriptor.h

```diff
@@ -87,6 +90,8 @@ class Descriptor {
 
   virtual Result UpdateDescriptorSet(VkDescriptorSet descriptor_set) = 0;
   virtual void SendDataToGPUIfNeeded(VkCommandBuffer command) = 0;
```

Comment:
So, this was me, heh. I find Host/Device confusing so asked for CPU/GPU to make it clear where the memory was/is.

Reply:
Chatted with David about this and I was wrong. These should be Host/Device because there are cases where Host/Device is CPU/GPU and cases where it's CPU/CPU. So Host/Device is the more unique and correct naming.

Sorry about that.